rpcs3/rpcs3qt/tooltips.h

```diff
@@ -73,11 +73,11 @@ class Tooltips : public QObject
 
 		// debug
 
-		const QString ppu_debug                    = tr("Never use this.");
-		const QString spu_debug                    = tr("Never use this.");
-		const QString set_daz_and_ftz              = tr("Never use this.");
-		const QString accurate_getllar             = tr("Never use this.");
-		const QString accurate_putlluc             = tr("Never use this.");
+		const QString ppu_debug                    = tr("Creates PPU logs.\nOnly useful to developers.\nNever use this.");
+		const QString spu_debug                    = tr("Creates SPU logs.\nOnly useful to developers.\nNever use this.");
+		const QString set_daz_and_ftz              = tr("Sets special MXCSR flags to debug errors in SSE operations.\nOnly used in PPU thread when it's not precise.\nOnly useful to developers.\nNever use this.");
+		const QString accurate_getllar             = tr("Accurately processes SPU MFC_GETLLAR operation.\nOnly useful to developers.\nNever use this.");
+		const QString accurate_putlluc             = tr("Accurately processes SPU MFC_PUTLLUC operation.\nOnly useful to developers.\nNever use this.");
```

Comment:
It's not true, those two are not only useful for developers. Keep the first sentence only.

Reply:
Which one exactly? You've selected 4 of them.

Reply:
last two.

Reply:
I mean these settings useful sometimes, it was "Never use this" as the developers were too lazy finding the accurate description for it, but yours is good enough for now.